components/FullEventCard/FullEventCard.js

```diff
@@ -8,6 +8,14 @@ export default function FullEventCard({ route }) {
     Alert.alert('Welcome to the Party! 🤙')
   }
 
+  const findDisplayTime = (eventTime) => {
+    let newTime = parseInt(eventTime.toString().substring(0,2));
+    let hours = ((newTime + 11) % 12) + 1;
+    let ampm = newTime > 11 ? 'pm' : 'am';
+    let minutes = eventTime.toString().substring(2);
+    return hours + ':' + minutes + ampm;
+  }
+
```

Comment:
Excellent work converting 24hr time to 12hr time! This will be useful throughout our app!